common/com_model.h

```diff
@@ -109,7 +109,7 @@ typedef struct glpoly_s
 	struct glpoly_s	*chain;
 	int		numverts;
 	int		flags;          		// for SURF_UNDERWATER
-	float		verts[4][VERTEXSIZE];	// variable sized (xyz s1t1 s2t2)
+	float		(*verts)[VERTEXSIZE];	// variable sized (xyz s1t1 s2t2)
```

Comment:
Maybe just not breaking binary compability?
On Mar 25, 2016 7:52 AM, "nekonomicon" <notifications@github.com> wrote:

> In common/com_model.h
> <https://github.com/SDLash3D/xash3d/pull/111#discussion_r57419133>:
>
> > @@ -109,7 +109,7 @@ typedef struct glpoly_s
> >  	struct glpoly_s	*chain;
> >  	int		numverts;
> >  	int		flags;          		// for SURF_UNDERWATER
> > -	float		verts[4][VERTEXSIZE];	// variable sized (xyz s1t1 s2t2)
> > +	float		(*verts)[VERTEXSIZE];	// variable sized (xyz s1t1 s2t2)
>
> May be use macro for non-windows as temporary solution?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/SDLash3D/xash3d/pull/111/files/00a8120579f776de96569a0b74d85e413a9a2b7e#r57419133>
>